src/parser/types.ts

```diff
@@ -446,7 +454,41 @@ export interface WebsiteComponent extends BaseIndividualComponent {
 }
 
 /**
- * The QuestionnaireComponent interface is used to define the properties of a questionnaire component. A QuestionnaireComponent is used to render questions with different response types. The response types are also defined with these documentation. The main use case of this component type is to ask participants questions, without using markdown, websites, images, etc.
+ * A QuestionnaireComponent is used to render simple questions that require a response. The main use case of this component type is to ask participants questions when you don't need to render a stimulus. Please note, that even though we're not using a stimulus, the responses still require a `location`. For example this could be used to collect demographic information from a participant using the following snippet:
+ *
+ * ```js
+  {
+    "type": "questionnaire",
+    "response": [
+      {
+        "id": "gender",
+        "prompt": "Gender:",
+        "required": true,
+        "location": "belowStimulus",
+        "type": "checkbox",
+        "options": [
+          {
+            "label": "Man",
+            "value": "Man"
+          },
+          {
+            "label": "Woman",
+            "value": "Woman"
+          },
+          {
+            "label": "Genderqueer",
+            "value": "Genderqueer"
+          },
+          {
+            "label": "Third-gender",
+            "value": "Third-gender"
```

Comment:
Typically, I think this is either "Third gender" or "Third-gendered". I know it's just an example, but just double check that.

Reply:
I think this might just be confusing grammar. I copied the survey example from Queer in AI so I think it should be good.